Final Project - C.txt

```diff
@@ -0,0 +1,99 @@
+		future assaults.  	
+		
+		- Scavenge: The player will have the choice of what they want to look for,
+		within the five supply groups. This is risker than most of the other actions.
+[]		
+		- "Farming": This is general category in which food production and preparation falls. While 
+		this is not really farming, it is attempting to focus on getting more food and is not as risky as scavenging.
+		
+		- Tinker: This allows for growth of the group members. This, depending on the successfullness, can give your group members 
+		permanent upgrades. Additionally, it can allow for weapon fabrication or car repairs. This element is entirely random - otherwise
+		it would be too strong - except when the car is broken. Then, all tinkering actions are dedicating to attempting to repair it.
+		
+		- Assault: This is BY FAR the riskiest action in the game. Group members will almost certainly die. That being said,
+		it is definitely the strongest way to get a lot of resources.
+		
+		
+		Game Mechanics & Daily Events:
+	
+	An important element of this game/simulation will be random, daily events. One such mechanic will
+be the base assault, which can be safeguarded against by improving the group's base. Other daily events
+include dice rolls on being assaulted while scavenging, varying success rates for actions - especially
+when said action is done many times in succession. As time progresses, each turn the user takes being counted
+as one day, loot becomes scarcer and attacks become more frequent. Even if players play exceedingly well, unlucky dice rolls
+and daily events can ruin even a seemingly stable situation. Members of the player's party will die, and new party members are 
+probably among the rarest encounters. There are no win conditions for this game, as the point is to see how long you can survive. To quote
+Project Zomboid's developers, "These are the end times. This is how you died."
+	
+	In terms of game mechanics, there are 4 skills: Scavenging, Combat, Construction and Tinkering.
+At party creation, each character is allowed to have a speciality in any one of these 4. This will give a bonus to that character.
+These stats are used, in the case of combat, to roll against enemies, whose strength increases as the game progresses. If every member has a weapon,
+then the bonus from weapons is maximized; if not enough weapons are present, advantage is given on a percentage-based system. For all other skills,
+a success multipler is given. These are calculated by adding together all the current group member's deviation from the norm. There are very rare events
+that can give permanent stat upgrades, but do not rely on these.
+		
+	The game is lost if every party member dies. If there is not enough food, every action that the group takes has a -5%(subject to change) to every action per day
+that they have not eaten. If that modifier drops below 50%, the group will starve.	

```

Comment:
This is a hefty (and impressive) project!  Be sure to implement it in pieces such that you have a working project at any given stage.  While most of your concept implementations are able to be inferred, you should be sure to use arrays/strings and structures/enums in a useful and significant way.